califa/calibration/R3BCalifaCrystalCal2Cluster.cxx

```diff
@@ -90,45 +198,41 @@ void R3BCalifaCrystalCal2Cluster::SetParContainers()
 InitStatus R3BCalifaCrystalCal2Cluster::Init()
 {
     R3BLOG(info, "");
-    assert(!fCalifaClusterData); // in case someone calls Init() twice.
+    assert(!fCalifaHitData); // in case someone calls Init() twice.
 
     FairRootManager* ioManager = FairRootManager::Instance();
     R3BLOG_IF(fatal, !ioManager, "FairRootManager not found");
 
     fCrystalCalData = (TClonesArray*)ioManager->GetObject("CalifaCrystalCalData");
 
-    // Register output array
-    fCalifaClusterData = new TClonesArray("R3BCalifaClusterData");
-    ioManager->Register("CalifaClusterData", "CALIFA Hit", fCalifaClusterData, !fOnline);
+    fCalifaHitData = new TClonesArray("R3BCalifaClusterData");
+    ioManager->Register("CalifaClusterData", "CALIFA Hit", fCalifaHitData, !fOnline);
 
     fCalifaGeo = R3BCalifaGeometry::Instance();
+
+    fCalifaGeo->Init(fGeometryVersion);
+
     R3BLOG_IF(error, !fCalifaGeo->Init(fGeometryVersion), "Califa geometry not found");
 
-    // Determine CALIFA position with respect to target
-    if (fCalifaGeo->IsSimulation())
-    {
-        R3BLOG(info, "Simulation configuration.");
-        fCalifatoTargetPos = 2.0 * fTargetPos - fCalifaPos;
-    }
-    else
-    {
-        R3BLOG(info, "Analysis configuration.");
-        fCalifatoTargetPos = fTargetPos - fCalifaPos;
-    }
```

Comment:
For some reason someone checked with this fCalifaGeo -> IsSimulation() if this was a simulation run. By calling R3BCalifaGeometry::IsSimulation(true) a flag was set in R3BCalifaGeometry and something changed in the top volumes definition, so the 2 factor was needed. Do not ask me why. I just set the simulation flag here instead of in the geo and things seem to be more consistent with respect to the target position.  
 `fCalifatoTargetPos = fTargetPos - fCalifaPos;` is enough to set things properly.